daffodil-cli/src/it/scala/org/apache/daffodil/cliTest/TestCLIUdfs.scala

```diff
@@ -179,105 +225,159 @@ class TestCLIUdfs {
+      "daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd",
+    )
+    val classpath = udfClasspath(
+      "daffodil-udf/src/test/java/org/badudfs/evaluate/StringFunctions/",
+    )
 
     runCLI(args"-v parse -s $schema -r user_func1", classpath) { cli =>
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.Replace." +
-        " Missing evaluate method: urn:example:com:ext:badudfs:stringfunctions:replace")
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.FuncA." +
-        " Overloaded evaluate method: urn:example:com:ext:badudfs:stringfunctions:funcA")
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.FuncB." +
-        " Unsupported return type: void")
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.FuncC." +
-        " Unsupported parameter type(s): String[],int[]")
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.FuncD." +
-        " Unsupported parameter type(s): String[]")
-      cli.expectErr("[warn] User Defined Function ignored:" +
-        " org.badudfs.evaluate.StringFunctions.FuncE." +
-        " Unsupported return type: String[]")
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.Replace." +
+          " Missing evaluate method: urn:example:com:ext:badudfs:stringfunctions:replace",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncA." +
+          " Overloaded evaluate method: urn:example:com:ext:badudfs:stringfunctions:funcA",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncB." +
+          " Unsupported return type: void",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncC." +
+          " Unsupported parameter type(s): String[],int[]",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncD." +
+          " Unsupported parameter type(s): String[]",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncE." +
+          " Unsupported return type: String[]",
+      )
       cli.expectErr("[error] Schema Definition Error: Unsupported function: jsudf:replace")
-    } (ExitCode.UnableToCreateProcessor)
+    }(ExitCode.UnableToCreateProcessor)
   }
 
   /**
    * Tests the case when a function class:
    *   throws custom error on evaluate
    */
   @Test def test_UDFClass_CustomExceptionOnEvaluate(): Unit = {
```

Comment:
This is one of the two IT tests which are failing in CI.  This one fails with `net.sf.expectit.ExpectIOException: Expect operation fails (timeout: -1 ms) for matcher: contains('at org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse.evaluate(StringFunctionsProvider.scala:56)')`.  I've looked at the formatting changes and I don't think they've changed the string arguments, only the whitespace around them.  You'll have to debug the test to see what the text being matched against really contains.  I'm pretty sure the line number changed because the other test file got reformatted (the StringFunctions.Reverse file).  Simply take out the line number from the string argument at line 288 below.